docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -1,60 +1,64 @@
+- Workflow-related services. Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service), and WCF Data Services (formerly known as ADO.NET Data Services) are only available on .NET Framework. There are currently no plans to bring them to .NET 6.
 
 In addition to the technologies listed in the official [.NET roadmap](https://github.com/dotnet/core/blob/main/roadmap.md), other features might be ported to the new [unified .NET platform](https://devblogs.microsoft.com/dotnet/introducing-net-5/). You might consider participating in the discussions on GitHub so that your voice can be heard. And if you think something is missing, file a new issue in the [dotnet/runtime](https://github.com/dotnet/runtime/issues/new) GitHub repository.
 
-## Using a platform or API that doesn't support .NET 5
+## Using a platform or API that doesn't support .NET 6
 
-Some Microsoft and third-party platforms don't support .NET 5. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET 5. Most Azure SDK should eventually be ported to .NET 5/Standard but some might not for various reasons. You can see the available Azure SDKs in the [Azure SDK Latest Releases](https://azure.github.io/azure-sdk/releases/latest/index.html) page.
+Some Microsoft and third-party platforms don't support .NET 6. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET 6. Most Azure SDK should eventually be ported to .NET 6/Standard but some might not for various reasons. You can see the available Azure SDKs in the [Azure SDK Latest Releases](https://azure.github.io/azure-sdk/releases/latest/index.html) page.
 
-In the meantime, if any platform or service in Azure still doesn't support .NET 5 with its client API, you can use the equivalent REST API from the Azure service or the client SDK on .NET Framework.
+In the meantime, if any platform or service in Azure still doesn't support .NET 6 with its client API, you can use the equivalent REST API from the Azure service or the client SDK on .NET Framework.
+
+## Porting existing ASP.NET application to .NET 6
+
+.NET Core is a revolutionary step forward from .NET Framework. It offers a host of advantages over .NET Framework across the board from productivity to performance, from cross-platform support to developer satisfaction. If you are using .NET Framework and planning to migrate your application to .NET Core and later to .NET 6, then refer to this guide [Porting Existing ASP.NET Apps to .NET Core](https://docs.microsoft.com/dotnet/architecture/porting-existing-aspnet-apps/).
 
 ### Additional resources
 
 - **.NET fundamentals** \
   [https://docs.microsoft.com/dotnet/fundamentals](../../../fundamentals/index.yml)
 
-- **Porting Projects to .NET 5** \
+- **Porting Projects to .NET 6** \
```

Comment:
The video you are referencing is still .NET 5?

Reply:
Yes, correct. Removed the links.